----------------------------------------------
16:15:37 986 Class: Regexs
Method: RunPatternRegular
System.ArgumentException: parsing "(?<Time>[0-9][0-9]:[0-9][0-9:]:[0-9][0-9]) Error event: (?<Status>(^\d+{4}-0000))" - Nested quantifier {.
   at System.Text.RegularExpressions.RegexParser.ScanRegex()
   at System.Text.RegularExpressions.RegexParser.Parse(String re, RegexOptions op)
   at System.Text.RegularExpressions.Regex..ctor(String pattern, RegexOptions options, TimeSpan matchTimeout, Boolean useCache)
   at System.Text.RegularExpressions.Regex..ctor(String pattern, RegexOptions options)
   at Transaction_Statistical.Regexs.RunPatternRegular(String sString, String sReg, Dictionary`2& listResult) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 1582
----------------------------------------------

----------------------------------------------
16:25:29 437 Class: Regexs
Method: RunPatternRegular
System.ArgumentException: parsing "(?<Time>[0-9][0-9]:[0-9][0-9:]:[0-9][0-9]) Error event: (?<Status>([1-9]+|\w+-\d+)" - Not enough )'s.
   at System.Text.RegularExpressions.RegexParser.ScanRegex()
   at System.Text.RegularExpressions.RegexParser.Parse(String re, RegexOptions op)
   at System.Text.RegularExpressions.Regex..ctor(String pattern, RegexOptions options, TimeSpan matchTimeout, Boolean useCache)
   at System.Text.RegularExpressions.Regex..ctor(String pattern, RegexOptions options)
   at Transaction_Statistical.Regexs.RunPatternRegular(String sString, String sReg, Dictionary`2& listResult) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 1582
----------------------------------------------

----------------------------------------------
16:27:59 301 Class: Regexs
Method: RunPatternRegular
System.ArgumentException: parsing "(?<Time>[0-9][0-9]:[0-9][0-9:]:[0-9][0-9]) Error event: [^DEPOSIT&^WITHDRAWAL])" - Too many )'s.
   at System.Text.RegularExpressions.RegexParser.ScanRegex()
   at System.Text.RegularExpressions.RegexParser.Parse(String re, RegexOptions op)
   at System.Text.RegularExpressions.Regex..ctor(String pattern, RegexOptions options, TimeSpan matchTimeout, Boolean useCache)
   at System.Text.RegularExpressions.Regex..ctor(String pattern, RegexOptions options)
   at Transaction_Statistical.Regexs.RunPatternRegular(String sString, String sReg, Dictionary`2& listResult) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 1582
----------------------------------------------

----------------------------------------------
23:01:28 834 Class: <FindEventTransaction>d__32
Method: MoveNext
System.ArgumentOutOfRangeException: The added or subtracted value results in an un-representable DateTime.
Parameter name: months
   at System.DateTime.AddMonths(Int32 months)
   at System.DateTime.AddYears(Int32 value)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 727
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:01:35 128 Class: <FindEventTransaction>d__32
Method: MoveNext
System.ArgumentOutOfRangeException: The added or subtracted value results in an un-representable DateTime.
Parameter name: months
   at System.DateTime.AddMonths(Int32 months)
   at System.DateTime.AddYears(Int32 value)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 727
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:01:55 281 Class: <FindEventTransaction>d__32
Method: MoveNext
System.ArgumentOutOfRangeException: The added or subtracted value results in an un-representable DateTime.
Parameter name: months
   at System.DateTime.AddMonths(Int32 months)
   at System.DateTime.AddYears(Int32 value)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 727
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:01:56 485 Class: <FindEventTransaction>d__32
Method: MoveNext
System.ArgumentOutOfRangeException: The added or subtracted value results in an un-representable DateTime.
Parameter name: months
   at System.DateTime.AddMonths(Int32 months)
   at System.DateTime.AddYears(Int32 value)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 727
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:02:41 174 Class: <FindEventTransaction>d__32
Method: MoveNext
System.ArgumentOutOfRangeException: The added or subtracted value results in an un-representable DateTime.
Parameter name: months
   at System.DateTime.AddMonths(Int32 months)
   at System.DateTime.AddYears(Int32 value)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 727
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:02:42 068 Class: <FindEventTransaction>d__32
Method: MoveNext
System.ArgumentOutOfRangeException: The added or subtracted value results in an un-representable DateTime.
Parameter name: months
   at System.DateTime.AddMonths(Int32 months)
   at System.DateTime.AddYears(Int32 value)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 727
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:02:42 919 Class: <FindEventTransaction>d__32
Method: MoveNext
System.ArgumentOutOfRangeException: The added or subtracted value results in an un-representable DateTime.
Parameter name: months
   at System.DateTime.AddMonths(Int32 months)
   at System.DateTime.AddYears(Int32 value)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 727
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:02:43 826 Class: <FindEventTransaction>d__32
Method: MoveNext
System.ArgumentOutOfRangeException: The added or subtracted value results in an un-representable DateTime.
Parameter name: months
   at System.DateTime.AddMonths(Int32 months)
   at System.DateTime.AddYears(Int32 value)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 727
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:28 818 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:28 867 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:28 940 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:29 050 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:29 113 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:28 940 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:29 281 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:46:29 534 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:29 640 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:30 429 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:31 461 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:32 418 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:33 451 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:34 465 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:34 490 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:35 228 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:46:35 302 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:46:35 923 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:36 969 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:37 927 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:38 911 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:39 999 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:40 719 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:46:40 777 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:46:40 835 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:46:40 719 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:46:40 922 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:46:41 012 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:42 437 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:43 407 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:44 407 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:45 434 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:46 444 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:47 521 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:48 438 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:49 482 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:50 470 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:51 420 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:52 266 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:46:52 320 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:46:52 721 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:46:52 852 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:46:52 886 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:46:53 064 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:46:53 434 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:54 431 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:55 319 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:56 471 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:57 458 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:58 470 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:46:59 489 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:00 423 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:01 460 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:03 529 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:04 531 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:05 461 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:06 566 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:08 191 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:08 192 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:09 150 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:09 150 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:09 159 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:09 151 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:09 150 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:09 755 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:09 755 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:10 546 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:10 546 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:10 546 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:10 827 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:10 546 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:10 546 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:10 770 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:10 546 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:10 870 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:11 031 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:11 068 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:10 974 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:11 960 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:12 953 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:13 985 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:14 912 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:15 940 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:17 001 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:17 949 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:18 971 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:19 949 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:20 942 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:21 922 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:22 977 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:23 978 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:24 979 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:25 954 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:26 960 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:27 967 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:29 035 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:30 015 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:30 956 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:31 992 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:32 966 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:34 003 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:35 023 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:35 958 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:36 957 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:37 985 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:39 912 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:40 998 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:41 934 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:42 955 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:43 973 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:44 968 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:45 924 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:46 949 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:47 993 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:48 938 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:49 977 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:50 919 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:51 928 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:52 931 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:53 979 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:54 970 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:56 020 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:57 027 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:57 917 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:58 929 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:47:59 973 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:00 996 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:01 958 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:02 993 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:03 957 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:04 942 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:05 978 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:06 918 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:07 981 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:08 931 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:09 958 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:55 793 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:55 878 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:55 958 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:55 984 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:55 794 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:56 105 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:55 799 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:56 492 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:56 493 Class: InitParametar
Method: Send_Error
System.IO.IOException: The process cannot access the file 'D:\expert.log' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.InternalAppendAllText(String path, String contents, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents)
   at Transaction_Statistical.InitParametar.Send_Error(String MsgError, String ClassName, String MethodName) in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 202
----------------------------------------------

----------------------------------------------
23:48:57 413 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:58 483 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:48:59 469 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:49:00 395 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:49:01 418 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:49:02 397 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

----------------------------------------------
23:49:03 415 Class: <FindEventTransaction>d__32
Method: MoveNext
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Transaction_Statistical.ReadTransaction.<>c__DisplayClass32_2.<FindEventTransaction>b__0() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 733
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Transaction_Statistical.ReadTransaction.<FindEventTransaction>d__32.MoveNext() in D:\05-TEMP\Source\GitHub\ToolTCB\Transaction Statistical\Class\Defind.cs:line 713
----------------------------------------------

